tests/kuttl/tests/client/01-deploy-client.yaml

```diff
@@ -0,0 +1 @@
+../../../../config/samples/client_v1beta1_openstackclient.yaml
```

Comment:
This test is not passing for me in CRC because memcached is missing and is blocking keystone for coming up. We should either deploy a memcached sample as part of the deploy or add `memcached_deploy` as a dependency of `infra-kuttl`

Reply:
Hmm. That's true. Let me consider how I can resolve the circular dependencies between infra and keystone. (client -> keystone -> memcached)

It's wired I didn't catch it when I tested it in my local.

Reply:
I've pushed https://github.com/openstack-k8s-operators/install_yamls/pull/399 which would resolve the problem.